napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -265,22 +264,6 @@ def is_vertical(self):
         return self.size().height() > self.size().width()
 
     def _on_visibility_changed(self, visible):
-        from napari.viewer import Viewer
```

Comment:
curious what this code is being replaced with, and why, and why we still need the `_on_visibility_changed` function here?

Reply:
AFAICT it is setting the 'checked' status of actions in the plugins menu, which is handled with app-model actions (the 'toggled' field).

This function is connect to `visibilityChanged` event of `QtViewerDockWidget` and seems to just deal with the widget title (`QtCustomTitleBar`) now?

Reply:
Ok great, thanks @lucyleeow!